api/provider/aws/defaults.go

```diff
@@ -121,3 +123,19 @@ const DEFAULT_LB_ROLE_POLICY_TEMPLATE = `{
         }
     ]
 }`
+
+var DEFAULT_CREATE_LB_PORT_MODEL = []models.Port{
```

Comment:
I was thinking about this today - why do we have a default port? It doesn't make much sense to me. It's pretty rare that you actually want a `80:80/tcp` port... seems better to just not have a default. It also make a ton of sense when you think of it from a terraform configuration perspective.

Reply:
That makes sense, but should I encapsulate that in a different PR/open a different issue for fixing that specific problem?